packages/xo-server/src/xo-mixins/backups-ng-logs.js

```diff
@@ -129,6 +129,13 @@ export default {
             data: data.data,
             message,
           })
+      } else if (event === 'task.info') {
+        const parent = started[data.taskId]
+        parent !== undefined &&
+          (parent.infos || (parent.infos = [])).push({
+            data: data.data,
+            message,
+          })
```

Comment:
In our case yes because the message is used to identify the task. But it will not be pertinent if we want to add an explicit message to notify a user about an event.